src/pages/groupComparison/groupSelector/GroupSelector.tsx

```diff
@@ -14,6 +14,7 @@ import { action, computed, makeObservable, observable } from 'mobx';
 import { ComparisonGroup } from '../GroupComparisonUtils';
 import CollapsedGroupsButton from './CollapsedGroupsButton';
 import { submitToStudyViewPage } from 'pages/resultsView/querySummary/QuerySummaryUtils';
+import { group } from 'yargs';
```

Comment:
seems like this is a mistake?

Reply:
Yes, I didn't even see it imported before my commit